lib/ocpp/v201/charge_point.cpp

```diff
@@ -1919,9 +1943,9 @@ void ChargePoint::transaction_event_req(const TransactionEventEnum& event_type,
         remote_start_id_per_evse.erase(it);
     }
 
-    if (event_type == TransactionEventEnum::Started and (!evse.has_value() or !id_token.has_value())) {
-        EVLOG_error << "Request to send TransactionEvent(Started) without given evse or id_token. These properties "
-                       "are required for this eventType \"Started\"!";
+    if (event_type == TransactionEventEnum::Started and (!evse.has_value())) {
```

Comment:
One could argue we could remove this check too since this is only for specific cases. If you would configure TxStart to be Authorized then you don't necessarily know which EVSE that is going to be.